cmd/node/node.go

```diff
@@ -80,6 +83,10 @@ const (
 	PoetListenerLogger   = "poetListener"
 	NipstBuilderLogger   = "nipstBuilder"
 	AtxBuilderLogger     = "atxBuilder"
+	GrpcLogger           = "grpc"
+	NodeLogger           = "node"
```

Comment:
Let's pick a consistent naming convention for the loggers. I like `meshService` for the mesh service, how about `nodeService` for this one?

Reply:
Deal.